docs/markdown/home.md

```diff
@@ -22,3 +22,5 @@ Thanks for visiting the documentation for [angularjs-portal](https://github.com/
 #### Integration with uPortal
 + Overview
 + Customizations/Setup
+
+[uw-frame docs]: http://uw-madison-doit.github.io/uw-frame/
```

Comment:
Ohhh, i see what you did there. Odd syntax, confusing almost. May I suggest we just have it inline?